code/LINDA/LINDA_fire.dm

```diff
@@ -152,18 +152,18 @@
 // Garbage collect itself by nulling reference to it
 
 /obj/effect/hotspot/proc/Kill()
-	air_master.hotspots -= src
-	DestroyTurf()
-	qdel(src)
+	PlaceInPool(src)
```

Comment:
the qdel hint means that it will place it in the pool regardless, and qdel should be the standard for everything- not using an arbitrary proc.